apps/judicial-system/web/src/routes/Judge/Overview/Overview.treat.ts

```diff
@@ -9,3 +9,14 @@ export const infoSection = style({
   padding: `${theme.spacing[5]}px 0`,
   borderTop: `2px solid ${theme.color.purple100}`,
 })
+
+export const createGoProCaseContainer = style({
```

Comment:
I think it would be better not to use GoPro in class names and use something like 'createCourtCaseContainer' instead.